config/application.rb

```diff
@@ -14,5 +14,14 @@ class Application < Rails::Application
     # Settings in config/environments/* take precedence over those specified here.
     # Application configuration should go into files in config/initializers
     # -- all .rb files in that directory are automatically loaded.
+
+    Dotenv.load  # We need this to perform the checks below.
```

Comment:
Oh. Hmmm. I'm not sure I love this. I'll get back to you but I feel like loading ENV outside of the normal way is asking for trouble longterm. The `dotenv` gem usually just loads without this anyway so it's possible this is not necessary?